pkg/inject/inject.go

```diff
@@ -45,12 +46,21 @@ const (
 	envDestinationContext         = "LINKERD2_PROXY_DESTINATION_CONTEXT"
 	envDestinationProfileSuffixes = "LINKERD2_PROXY_DESTINATION_PROFILE_SUFFIXES"
 	envDestinationSvcAddr         = "LINKERD2_PROXY_DESTINATION_SVC_ADDR"
-
-	envIdentityDisabled = "LINKERD2_PROXY_IDENTITY_DISABLED"
-	identityDisabledMsg = "Identity is not yet available"
+	envDestinationSvcName         = "LINKERD2_PROXY_DESTINATION_SVC_NAME"
 
 	// destinationAPIPort is the port exposed by the linkerd-destination service
 	destinationAPIPort = 8086
+
+	envIdentityDisabled     = "LINKERD2_PROXY_IDENTITY_DISABLED"
+	envIdentityDir          = "LINKERD2_PROXY_IDENTITY_DIR"
+	envIdentityLocalName    = "LINKERD2_PROXY_IDENTITY_LOCAL_NAME"
+	envIdentitySvcAddr      = "LINKERD2_PROXY_IDENTITY_SVC_ADDR"
```

Comment:
Is this supposed to be consumed by the proxy? I don't see it in the [linkerd-proxy](https://github.com/linkerd/linkerd2-proxy/search?q=LINKERD2_PROXY_IDENTITY_SVC_ADDR&unscoped_q=LINKERD2_PROXY_IDENTITY_SVC_ADDR)  repo.

Reply:
Never, I think I found [it](https://github.com/linkerd/linkerd2-proxy/blob/2064900a0f89e84ece0bae96e160d5dbee2af848/src/app/config.rs#L789).